### ISSUE TITLE
Volumes of ARRR in iOS wallet don’t match volume of ARRR in Desktop Lite Chest 

### ISSUE THREAD

**Issue by Tyerh87** [Meshbits/pirate-chain-ios-wallet#75]
Volumes of ARRR in iOS wallet don’t match volume of ARRR in Desktop Lite Chest. Looking at the history of transactions it seems to be missing roughly a years history. 

Not sure what to try at this stage. Any suggestions? 

Thanks 
Tye

**Comment by satindergrewal** [comment]
Yes, try using earlier block height birthday to rescan your wallet if missing transactions and funds in calculated and scanned transactions with the birthday height you chose to setup wallet with. You can rescan your wallet with different block height birthday from settings of app.

**Comment by Tyerh87** [comment]
Thanks this seems to make sense. How would I know what the correct block height birthday I chose was? I can’t remember if I’m honest.

**Comment by satindergrewal** [comment]
If you don't know, the alternative will be to pick as earlier block height birthday you can. That will definitely work. The only downside is you have to keep your app open for longer time and don't let the app go in background or in sleep mode. We actually had this issue solved, unfortunately apple policy made us reverse our solution. Hope this clears and helps.
